src/server_manager/electron_app/release/notarize.js

```diff
@@ -19,7 +19,8 @@ const {notarize} = require('electron-notarize');
 // and `APPLE_PASSWORD`, the password to the account.
 exports.default = async function(context) {
   const {electronPlatformName, appOutDir} = context;
-  if (electronPlatformName !== 'darwin') {
+  if (electronPlatformName !== 'darwin' || !process.env.CSC_LINK) {
```

Comment:
What's the CSC_LINK env var? Maybe add a comment?

Reply:
Done in 316d69d.